<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorHueSaturation</class>
 <widget class="QDialog" name="ColorHueSaturation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>色调-饱和度</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>选择要修改的色系</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>全图</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>红色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>黄色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>绿色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>青色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>蓝色</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>品红</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>覆盖：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_overlap">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_overlap">
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>修改选中的颜色</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>色调：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_hue">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_hue">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>亮度：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_lightness">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_lightness">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>饱和度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_saturation">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_saturation">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset_color">
          <property name="text">
           <string>重置颜色</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_overlap</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_overlap</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_overlap</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_overlap</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_hue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_hue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_hue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_hue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_lightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_lightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_lightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_lightness</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_saturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_saturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_saturation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_saturation</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>ColorHueSaturation</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>ColorHueSaturation</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
